src/plugins/discover/server/ui_settings.ts

```diff
@@ -286,5 +286,6 @@ export const getUiSettings: (docLinks: DocLinksServiceSetup) => Record<string, U
         'The maximum height that a cell in a table should occupy. Set to 0 to disable truncation.',
     }),
     schema: schema.number({ min: 0 }),
+    requiresPageReload: true,
```

Comment:
this leads to a toast messaging the user to reloa
![Bildschirmfoto 2022-04-13 um 12 14 16](https://user-images.githubusercontent.com/463851/163158565-3b09edb1-a570-47be-97e9-e1f68233f17f.png)
d